osf/metrics.py

```diff
@@ -325,3 +326,139 @@ def get_latest_institution_project_document(cls, institution):
+               'doc_count': 4,
+               'rejected': {'doc_count': 1},
+               'submissions': {'doc_count': 1},
+               'not_embargoed_but_accepted': {'doc_count': 1},
+               'withdrawn': {'doc_count': 0},
+               'transitions_without_comments': {'doc_count': 0},
+               'embargoed': {'doc_count': 0},
+               'transitions_with_comments': {'doc_count': 4}
+               }]
+        }
+        :return: dict
+        """
+        search = cls.search()
+
+        return search.update_from_dict({
+            'aggs': {
+                'providers': {
+                    'terms': {
+                        'field': 'provider_id'
+                    },
+                    'aggs': {
+                        'transitions_without_comments': {
+                            'missing': {
+                                'field': 'comment'
+                            }
+                        },
+                        'transitions_with_comments': {
+                            'filter': {
+                                'exists': {
+                                    'field': 'comment'
+                                }
+                            }
+                        },
+                        'submissions': {
+                            'filter': {
+                                'match': {
+                                    'trigger': {
+                                        'query': 'submit'
+                                    }
+                                }
+                            }
+                        },
+                        'accepted_with_embargo': {
+                            'filter': {
+                                'match': {
+                                    'to_state': {
+                                        'query': 'embargo'
+                                    }
+                                }
+                            }
+                        },
+                        'accepted_without_embargo': {
+                            'filter': {
+                                'match': {
+                                    'to_state': {
+                                        'query': 'accepted'
+                                    }
+                                }
+                            }
+                        },
+                        'rejected': {
+                            'filter': {
+                                'match': {
+                                    'trigger': {
+                                        'query': 'reject'
+                                    }
+                                }
+                            }
+                        },
+                        'withdrawn': {
+                            'filter': {
+                                'match': {
+                                    'trigger': {
+                                        'query': 'reject'
```

Comment:
A withdrawn registration would have a trigger of `accept_withdrawal`, and we may also want `force_withdraw` here?